controller/controller.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/sky-uk/feed/k8s"
 	"github.com/sky-uk/feed/util"
 	v1 "k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/apis/extensions/v1beta1"
```

Comment:
Might need to use `apps/v1` package for deployments now that we are on 1.10 k8s.